stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messagepreview/MessagePreviewView.kt

```diff
@@ -78,7 +78,7 @@ public class MessagePreviewView : FrameLayout {
                 )
             )
         } else {
-            message.user.name.bold()
+            message.user.name.bold(ignoreCase = true)
```

Comment:
I don't think we need a change on this line, we're just making this entire string bold, there's no matching involved